flumine/backtest/simulated.py

```diff
@@ -84,7 +84,18 @@ def place(
                         size,
                         runner.ex.available_to_back,
                     )
-                    return self._create_place_response(bet_id)
+                    if self._should_expire():
+                        order_status = "EXPIRED"
+                        self.order.expired()
```

Comment:
Technically we shouldn't call this here (all examples below as well), as during live execution we use the `instruction_report_status` only, it then becomes execution complete during the order cycle as `size_cancelled == size`